### ISSUE TITLE
1.1.0 FTBFS

### ISSUE THREAD

**Issue by CameronNemo** [containers/libpod#2459]
/kind bug

**Steps to reproduce the issue:**

1. Build with tags: `seccomp apparmor containers_image_ostree_stub btrfs_noversion exclude_graphdriver_btrfs`

**Describe the results you received:**

```
src/github.com/containers/libpod/cmd/podman/rmi.go:39:8: undefined: iopodman.ImageNotFound
```

**Describe the results you expected:**

**Comment by mheon** [comment]
Looks like a Varlink issue - all our CI presently builds varlink enabled, so we didn't catch this.

Need to fix the issue and then get our CI to also build varlink disabled so it doesn't reoccur.

**Comment by mheon** [comment]
Opened #2465 to fix the build and #2464 to add CI to catch this

**Comment by CameronNemo** [comment]
Thank you for the fast response. I will give this a try in ~10-12 hours.
